examples/stdlib/list/countIf.effekt

```diff
@@ -0,0 +1,29 @@
+module examples/pos/list/countIf
+
+import list
+
+def main() = {
+
+        val empty: List[String] = []
+
+        println(empty.countIf{ x => x.length() == 2}) // expects 0
+
+
+
+        val listStr_none: List[String] = ["A","B","C"]
+
+        println(listStr_none.countIf{ x => x.length() == 2}) // expects 0
```

Comment:
Same here, please use two spaces for indentation, "camelCase" for identifiers, semantic names, and fewer newlines:
```suggestion
  val singleLetters: List[String] = ["A", "B", "C"]
  println(singleLetters.countIf { x => x.length() == 2 }) // expects 0
```